assets/main.js

```diff
@@ -4,14 +4,29 @@ $(document).ready( function(){
 
         var userCity = "";
 
-        var APIKEY = "376fffcd91763ad35ed350f2f78f8296";
+        var openweathermapAPIKEY = "376fffcd91763ad35ed350f2f78f8296";
 
-        var queryURL = "https://api.openweathermap.org/data.2.5/weather?q=" + userCity +"$appid=" + APIKEY;
+        var openweathermapQueryURL = "https://api.openweathermap.org/data.2.5/weather?q=" + userCity +"$appid=" + openweathermapAPIKEY;
 
-        $.ajax((
-            url: queryURL,
+        $.ajax({
+            url: openweathermapQueryURL,
             method: "GET"
-        )).then(function(response) {
+        }).then(function(response) {
+            console.log(response);
+        })
+    }
+    function getDestination(){
+
+        var destCity ="";
+
+        var kayakAPIKEY = "95faa2613dmsh9cf38f16b3fd33bp1f9e0djsnbdbe12d99b9f"
+
+        var kayakQueryURL = "https://apidojo-kayak-v1.p.rapidapi.com/locations/search?q=" + destCity + "$appid=" + kayakAPIKEY;
+
+        $.ajax({
+            url: kayakQueryURL,
+            method: "GET",
+        }).then(function(response) {
             console.log(response);
         })
```

Comment:
$.ajax({
      url: "https://apidojo-kayak-v1.p.rapidapi.com/locations/search?where=" + destCity,
      method: "GET",
      headers: {
        "X-RapidAPI-Key": kayakAPIKEY
      }
    }).then(function(response) {
      console.log(response)
    });

Reply:
Yeah this looks a lot more like it.